x-pack/plugin/core/template-resources/src/main/resources/connectors-deleted.json

```diff
@@ -0,0 +1,307 @@
+{
```

Comment:
Is it possible to have a test that checks that both deleted and non-deleted indices have the same schema?

Additionally, does it make sense to make a lot of properties here searchable? Or we wanna have filters on deleted connectors too?